src/components/tooltip/Tooltip.tsx

```diff
@@ -332,17 +339,23 @@ class Tooltip extends React.Component<TooltipProps, State> {
                 onContextMenu={this.handleTooltipEvent}
                 onKeyPress={this.handleTooltipEvent}
             >
-                <div role="tooltip" aria-live="polite" aria-hidden={isLabelMatchingTooltipText}>
+                <div
+                    role={tooltipRole}
+                    aria-live="polite"
+                    aria-hidden={isLabelMatchingTooltipText}
+                    data-testid="tooltip"
```

Comment:
Let's use `bdl-Tooltip`?

Reply:
@jstoffan I just updated this PR with your comments